src/shisen_cpp/camera/node/camera_node.cpp

```diff
@@ -24,6 +24,7 @@
 #include <memory>
 #include <nlohmann/json.hpp>
 #include <shisen_cpp/camera/node/camera_node.hpp>
+#include "jitsuyo/config.hpp"
```

Comment:
Should use angled brackets:
```suggestion
#include <jitsuyo/config.hpp>
```